onchainaddressbook/create_onchain_address_book_entry.go

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package onchainaddressbook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/coinbase-samples/core-go"
+	"github.com/coinbase-samples/prime-sdk-go/client"
+	"github.com/coinbase-samples/prime-sdk-go/model"
+)
+
+type CreateOnchainAddressBookEntryRequest struct {
+	AddressGroup *model.OnchainAddressGroup `json:"address_group"`
+	PortfolioId  string                     `json:"portfolio_id"`
+}
+
+type CreateOnchainAddressBookEntryResponse struct {
+	ActivityId         string                                `json:"activity_id"`
+	ActivityType       string                                `json:"activity_type"`
```

Comment:
Should this be an enum?

Reply:
We haven't done this so far with the APIs, but I agree that we should refactor in the future.

Reply:
Added